packages/lisk-p2p/src/validators.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { onlyDigits, validateIp, validatePort } from './utils';
+
+interface ResponsePeerObject {
+	readonly height: string | number;
```

Comment:
why is this expecting string or number? shouldn't it be one way or another?

Reply:
From the response, we can expect either number or string, and both could be valid, but after we sanitize the response then we instantiate peer object with the port as number type. Or should we only use, one type?

Reply:
addressed